config/environments/production.rb

```diff
@@ -134,4 +134,19 @@
 
   # Add the GraphQL probe for Skylight.
   config.skylight.probes << "graphql"
+
+  # Add throttling to application
+  rules = [
+    { method: "POST", limit: 10 },
+    { method: "POST", path: "/graphql", limit: 10 }]
+
+  require 'dalli'
+
+  cache = if ENV['MEMCACHEDCLOUD_SERVERS'].present?
+    Dalli::Client.new((ENV['MEMCACHEDCLOUD_SERVERS']).split(","),
+      { :username => ENV['MEMCACHEDCLOUD_USERNAME'],
+        :password => ENV['MEMCACHEDCLOUD_PASSWORD'] })
+  end
```

Comment:
@mahesh-krishnakumar Shouldn't this `if` have an `else`?

Reply:
@harigopal Had tested it by passing a `nil`. No side effects were seen. As good as having no caching mechanism for rate-limiting.
https://github.com/dryruby/rack-throttle/blob/b19ec4c22f8ecb10574acc866a3876ab1b57068b/lib/rack/throttle/limiter.rb#L88